src/camera_streaming/config/webrtc.yaml

```diff
@@ -0,0 +1,10 @@
+webrtc_node:
+  ros__parameters:
+    camera_name:
+    - "Drive"
+    - "Arm"
+    camera_path:
+    - "/dev/v4l/by-id/usb-Sonix_Technology_Co.__Ltd._USB_2.0_Camera_SN5100-video-index0"
```

Comment:
Is this camera under the v4l dir because of the compression rather than like video2? Its more than fine for now, maybe once we get a full list of the cameras were going to use and plug them all in to link them to a more descriptive name under like /dev/rovercams/arm,drive etc

Reply:
This is a symlink that points to /dev/video<n>. This is unique to the camera model but might run into trouble if we use multiple identical cameras. If we want to add udev rules to create custom symbolic links we can do so.